src/Spice86.Core/Emulator/Devices/Input/Keyboard/Keyboard.cs

```diff
@@ -46,13 +50,43 @@ public class Keyboard : DefaultIOPortHandler {
     public override byte ReadByte(int port) {
         byte? scancode = LastKeyboardInput.ScanCode;
         if (scancode == null) {
-            return 0;
+            scancode = 0;
+        }
+        
+        return port switch {
+            KeyboardPorts.IOPort => scancode.Value,
+            // keyboard not locked, self-test completed.
+            KeyboardPorts.StatusRegister => 0x14,
```

Comment:
Since each bit has a meaning, shouldnt we write this like that? Maybe in the future we could have variable bits with a more refined implementation.
```
const byte SYSTEM_TEST_STATUS_MASK = 1<<2;
const byte KEYBOARD_ENABLE_STATUS_MASK = 1<<4;
KeyboardPorts.StatusRegister => SYSTEM_TEST_STATUS_MASK | KEYBOARD_ENABLE_STATUS_MASK;

```

Reply:
Done.